components/actiongroup/metadata/mods.md

```diff
@@ -2,9 +2,6 @@
 | ---------------------------------------------- |
 | `--mod-actiongroup-border-radius`              |
 | `--mod-actiongroup-border-radius-reset`        |
-| `--mod-actiongroup-button-spacing-reset`       |
 | `--mod-actiongroup-gap-size-compact`           |
-| `--mod-actiongroup-horizontal-spacing-compact` |
 | `--mod-actiongroup-horizontal-spacing-regular` |
-| `--mod-actiongroup-vertical-spacing-compact`   |
```

Comment:
Is it ok to remove these mods since this is a breaking change, or is there a different way we need to do this?

Reply:
Yes, I think this is an ok change to make!